webapp/advantage/ua_contracts/api.py

```diff
@@ -108,6 +108,20 @@ def get_account_users(self, account_id: str):
 
         return users
 
+    def put_account_user_role(
+        self,
+        account_id: str,
+        user_role_request: dict,
+    ):
+        self._request(
+            method="put",
+            path=f"v1/accounts/{account_id}/user-role",
+            json=user_role_request,
+            error_rules=["default"],
```

Comment:
If I understand correctly, this will make the endpoint return `UAContractsAPIErrorView`/`UAContractsAPIError` which returns a standard python HTTPError? Does this mean that there's only a single possible error that this endpoint will return?

I'm trying to list different errors and map them to error messages.